CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Version 3.0.0
+
+- Replace `libc` in all backends with the `rustix` crate (#108).
+- Use `tracing` instead of `log` for telemetry (#119).
```

Comment:
Personally, I prefer "logging" to "telemetry" here. While "telemetry" can mean "from remote", that's not what we're doing.